src/ngraph/validation_util.hpp

```diff
@@ -103,5 +103,17 @@ namespace ngraph
                                    const AxisSet& shrink_axis_mask,
                                    const AxisSet& ellipsis_mask);
 
-    std::size_t normalize_axis(const Node* node, std::int64_t axis, std::int64_t tensor_rank);
+    int64_t normalize_axis(const Node* node, std::int64_t axis, std::int64_t tensor_rank);
```

Comment:
Please add documentation for all of these functions.

Reply:
Done